x-pack/plugins/apm/server/lib/alerts/action_variables.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const apmActionVariables = {
```

Comment:
Extracted the action variables and their descriptions to avoid duplication and improve readability of alerts.